File: 2021 maj/python/1.py
with open("instrukcje.txt", "r") as f:
    inst = f.read().strip().split('\n')

ans = 0
for line in inst:
    s, c = line.split()
    if s == "DOPISZ":
        ans += 1
    elif s == "USUN":
        ans -= 1
print(ans)

File: 2021 maj/c++/1.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main()
{
    ifstream in("instrukcje.txt");
    
    string s;
    char c;
    int ans = 0;
    while (in >> s >> c)
    {
        if (s == "DOPISZ")
            ans++;
        else if (s == "USUN")
            ans--;
    }
    cout << ans << '\n';
}